cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataHttpHandlerTest.java

```diff
@@ -97,18 +97,18 @@ public void testProperties() throws IOException {
     Assert.assertEquals(expected, searchProperties);
 
     // test prefix search for service
-    searchProperties = searchMetadata(Id.Namespace.DEFAULT.getId(), "sKey:s", "ALL");
+    searchProperties = searchMetadata(Id.Namespace.DEFAULT.getId(), "sKey:s*", "ALL");
     expected = ImmutableSet.of(
       new MetadataSearchResultRecord(pingService, MetadataSearchTargetType.PROGRAM)
     );
     Assert.assertEquals(expected, searchProperties);
 
     // search without any target param
-    searchProperties = searchMetadata(Id.Namespace.DEFAULT.getId(), "sKey:s", null);
+    searchProperties = searchMetadata(Id.Namespace.DEFAULT.getId(), "sKey:s*", null);
```

Comment:
what will happen without the ``*``? Can you add a verification for that? It should return an empty set also, right?